lib/ClangImporter/ClangImporter.cpp

```diff
@@ -807,13 +807,8 @@ ClangImporter::getOrCreatePCH(const ClangImporterOptions &ImporterOptions,
   if (!PCHFilename.hasValue()) {
     return None;
   }
-  if (!isExplicit && !canReadPCH(PCHFilename.getValue())) {
-    SmallString<256> Message;
-    llvm::raw_svector_ostream OS(Message);
```

Comment:
I'm a little confused about what this temporary stream was doing before; looks like dead code?